README.MD

```diff
@@ -1934,39 +1935,55 @@ aem {
 Files section works in a same way as in [instance satisfy task](#task-instancesatisfy).
 For more details see AEM [File Install Provider](https://helpx.adobe.com/experience-manager/6-5/sites/deploying/using/custom-standalone-install.html#AddingaFileInstallProvider) documentation.
 
-##### Extracted instance files configuration (optional)
+##### Customization of extracted instance files (optional)
 
-Plugin allows to override or provide extra files to local AEM instance installations.
-This behavior is controlled by:
+The plugin allows us to override or provide extra files to local AEM instance installations.
+This behavior could be customized by `localInstance` section of plugin DSL:
 
 ```kotlin
 aem {
     localInstance {
-        rootDir = prop.string("localInstance.root")?.let { project.file(it) } ?: projectMain.file(".aem/instance")
-        overridesDir = File(configCommonDir, LocalInstance.FILES_PATH)
-        expandProperties = mapOf()
-        expandFiles = listOf(
+        rootDir = File(".instance") // path under which local instance files are stored
+        overridesDir = File("gradle/instance/local") // path with directories instance specific (common, author, publish) holding files that can override default instance files
```

Comment:
same issue

Reply:
`File(path) != file(path)`

`file()` is using Gradle file resolver / internal API